.github/workflows/ci.yaml

```diff
@@ -356,7 +357,10 @@ jobs:
       - name: Install CodeNotary
         shell: bash
         run: |
-          bash <(curl https://getvcn.codenotary.com -L)
+          mkdir -p "${{ github.action_path }}/bin"
```

Comment:
`github.action_path` only exists for actions, not in jobs.

Why do we even do this, it's not used?

Reply:
it is used on our tests to make sure it works correctly. I don't like to mockup that because a mocked test meaning nothing

Reply:
oh, so it can be used inside supervisor test :thinking: